pdcupdater/services.py

```diff
@@ -72,13 +75,19 @@ def old_composes(base_url):
 
 
 @pdcupdater.utils.with_ridiculous_timeout
-def fas_persons(base_url, username, password):
+def fas_persons(base_url, username, password, freeipa_backend):
     """ Return the list of users in the Fedora Account System. """
 
+    log.info("Connecting to Account System at %r" % base_url)
+    if freeipa_backend:
+        client = Client(url=base_url)
+        response = client.users.list_users().response().result
+        people = response.get('result', [])
+        return people
```

Comment:
We also also need to make sure that the user dictionary contains all the data that this app needs.

Reply:
After a quick grep, this app only seems to use the `username` key, and fasjson provides it, so it should be fine.